### DISCUSSION TITLE
测绘组件的editorOpts能不能自定义icon

### DISCUSSION THREAD

**Discussion by jiege15121313** [zouyaoji/vue-cesium#317] [Q&A]
![Uploading A4730C45-37A5-4792-A41C-B06FBA6A4848.png…]()

**Comment by zouyaoji** [comment]
可以的哈。

通过 `editorOpts` 配置。

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/defaultOpts.ts#L192
